ugvc/reports/report_utils.py

```diff
@@ -10,78 +10,10 @@
 import pandas as pd
 from IPython.display import Markdown, display
 from matplotlib.ticker import FormatStrFormatter
-from scipy.interpolate import interp1d
 
-from ugvc.utils.metrics_utils import get_histogram_from_sorter
-from ugvc.utils.misc_utils import set_pyplot_defaults
 from ugvc.utils.stats_utils import get_f1, get_precision, get_recall
 
 
-def plot_read_length_histogram(
```

Comment:
was moved to sorter_utils in ugbio_core:
https://github.com/Ultimagen/ugbio-utils/blob/cad8341a927c5641c0d063ace9e5326a2e1ec325/src/core/ugbio_core/sorter_utils.py#L139